src/store/slice/auth/types.ts

```diff
@@ -0,0 +1,58 @@
+export interface SignupPayloadI {
+  firstname: string;
+  lastname: string;
+  email: string;
+  password: string;
+}
+
+export interface SignupResponseI {
+  result: {
+    profilePicture: any;
```

Comment:
Remove any from here. Also define default values with all fields in SignupResponseI as null e.g. 

profilePic: string | null.